### PR TITLE
fix: Refactor dashboard links in leave policy

### PR DESCRIPTION
**Pull request by michellealva**
Changed placement of dashboard links in Leave Policy to make it more consistent and remove unwanted spacing.

**Before:**

image

**After:**

image

### CODE REVIEW

**Comment by michellealva** [comment]
@Mergifyio backport version-12-hotifx

File: erpnext/hr/doctype/leave_policy/leave_policy_dashboard.py [hunk 1]
```diff
@@ -8,13 +8,17 @@ def get_data():
 		},
 		'transactions': [
 			{
-				'items': ['Employee']
-			},
-			{
-				'items': ['Employee Grade']
+				'label': ('Employees'),
```
Comment (ruchamahabal) [inline erpnext/hr/doctype/leave_policy/leave_policy_dashboard.py]:
```suggestion
				'label': _('Employees'),
```
Make labels translatable

File: erpnext/hr/doctype/leave_policy/leave_policy_dashboard.py [hunk 2]
```diff
@@ -8,13 +8,17 @@ def get_data():
...
+				'items': ['Employee', 'Employee Grade']
 			},
 			{
+				'label': ('Leaves'),
```
Comment (ruchamahabal) [inline erpnext/hr/doctype/leave_policy/leave_policy_dashboard.py]:
```suggestion
				'label': _('Leaves'),
```

**Review by ruchamahabal — APPROVED** [review]

**Comment by ruchamahabal** [comment]
@Mergifyio backport version-12-hotfix

**Pull request merged**